lib/rubocop/cop/style/safe_navigation.rb

```diff
@@ -335,8 +338,14 @@ def matching_nodes?(left, right)
 
         def matching_call_nodes?(left, right)
           return false unless left && right.respond_to?(:call_type?)
+          return false unless left.call_type? && right.call_type?
+
+          # Compare receiver and method name, but ignore the difference between
+          # safe navigation (&.) and regular navigation (.)
+          left_receiver, left_method = left.children.first(2)
+          right_receiver, right_method = right.children.first(2)
```

Comment:
That's just my two cents:

```suggestion
          # Compare receiver and method name, but ignore the difference between
          # safe navigation method call (`&.`) and dot method call (`.`).
          left_receiver, left_method = left.children.take(2)
          right_receiver, right_method = right.children.take(2)
```